### PR TITLE
Feature/add react npm sample

### PR DESCRIPTION
**Pull request by kishorens**
Added react sample using npm packages

-Sample with bundled player
-Sample with modular player

### CODE REVIEW

File: react/modular-player/src/bitmovinPlayer.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import {Player} from 'bitmovin-player/modules/bitmovinplayer-core';
```
Comment (HalfbyteHeroes) [inline react/modular-player/src/bitmovinPlayer.js:2]:
```suggestion
import { Player } from 'bitmovin-player/modules/bitmovinplayer-core';
```

File: react/modular-player/src/bitmovinPlayer.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+import AbrModule from 'bitmovin-player/modules/bitmovinplayer-abr';
+import XmlModule from 'bitmovin-player/modules/bitmovinplayer-xml';
+import ContainerTSModule from 'bitmovin-player/modules/bitmovinplayer-container-ts';
+import ContainerMp4Module from 'bitmovin-player/modules/bitmovinplayer-container-mp4';
+import SubtitlesModule from 'bitmovin-player/modules/bitmovinplayer-subtitles';
+import SubtitlesCEA608Module from 'bitmovin-player/modules/bitmovinplayer-subtitles-cea608';
+import PolyfillModule from 'bitmovin-player/modules/bitmovinplayer-polyfill';
+import StyleModule from 'bitmovin-player/modules/bitmovinplayer-style';
+
+import {UIFactory} from 'bitmovin-player/bitmovinplayer-ui';
```
Comment (HalfbyteHeroes) [inline react/modular-player/src/bitmovinPlayer.js:16]:
```suggestion
import { UIFactory } from 'bitmovin-player/bitmovinplayer-ui';
```

**Review by HalfbyteHeroes — COMMENTED** [review]
I'm not familiar with React, so I can't give any advice on the concrete implementation itself, but the code looks good to me in general :+1:

File: react/bundled-player/src/bitmovinPlayer.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Player } from 'bitmovin-player';
+
```
[the remark this answers is not in the document]
Comment (kishorens) [reply react/bundled-player/src/bitmovinPlayer.js]: I added to differentiate player and UI import, removed it.

**Comment by kishorens** [comment]
Thanks for reviewing @HalfbyteHeroes

**Review by HalfbyteHeroes — APPROVED** [review]
Nice!

**Pull request merged**